sample/fill.html

```diff
@@ -0,0 +1,22 @@
+<!doctype html>
+<html>
+<head>
+<title>Form</title>
+</head>
+<body background="https://st2.depositphotos.com/2267983/7155/i/950/depositphotos_71559811-stock-photo-blurred-pastel-background.jpg">
+<form>
+<label for="name">Enter your name</label>
+<input type="text" id="name" name="name" value="name" required/> <br> <br>
+<label for="address">Where should we deliver the food?</label><br><br>
+<textarea id="address" name="address" rows="10" cols="39" required>
+</textarea><br><br>
+<label for="phone">Contact Number</label>
+<input type="text" id="phone" name="phone" value="+91" min=10 max=11 required/><br><br>
+
+<input type="submit" value="Submit"/>
+<input type="reset" value="Reset"/>
+
+<a href="Demo.html">Go Back</a>
```

Comment:
There should not be any top-level <a> tag ever. Wrap the text around using span or div.